plasmapy/physics/parameters.py

```diff
@@ -178,12 +179,13 @@ def Alfven_speed(B, density, ion="p"):
     except Exception:
         raise ValueError("Unable to find Alfven speed")
 
-    _check_relativistic(V_A, 'Alfven_speed')
-
     return V_A
 
 
-def ion_sound_speed(*ignore, T_e=0*units.K, T_i=0*units.K,
+@check_relativistic
+@check_quantity("T_e", can_be_negative=False)
+@check_quantity("T_i", can_be_negative=False)
```

Comment:
Is there a way this could be implemented so that you could specify both in one check_quantity call? I feel like that might be more readable than decorating the function three times.